openmicroscopy/openmicroscopy
# Remove epydoc targets/styles from the repository

With https://github.com/openmicroscopy/openmicroscopy/pull/2813 merged into `develop`, we moved to Sphinx to generate the Python API documentation. This PR fully removes the dependency on the `epydoc` module from the build system. 

To test this PR:
- check the documentation still builds as expected including the Python API
- additionally, http://ci.openmicroscopy.org/view/Experimental/job/OMERO-5.1-merge-win/ should now be able to build without the `epydoc` targer.

/cc @rleigh-dundee
